From 2290cf018e2fef96c737b4cfe3f330435ccbdc76 Mon Sep 17 00:00:00 2001
From: Luca Beltrame <lbeltrame@kde.org>
Date: Fri, 31 Mar 2017 17:44:22 +0200
Subject: Fix build with Qt 5.7

QPlatformSurfaceEvent is likely included by something else in Qt 5.8,
but it doesn't in Qt 5.7.

Observed in openSUSE Tumbleweed (which has Qt 5.7).

CCMAIL: hein@kde.org
---
 app/mainwindow.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/app/mainwindow.cpp b/app/mainwindow.cpp
index ea8ee8a..a7b0d89 100644
--- a/app/mainwindow.cpp
+++ b/app/mainwindow.cpp
@@ -54,6 +54,7 @@
 #include <QWhatsThis>
 #include <QWindow>
 #include <QtDBus/QtDBus>
+#include <QPlatformSurfaceEvent>
 
 #if HAVE_X11
 #include <QX11Info>
-- 
cgit v0.11.2

